CHANGELOG.md

```diff
@@ -3,6 +3,66 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.3.2
+
+_May 2, 2024_
+
+We'd like to offer a big thanks to the 8 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎁 Add "no data" and "loading" states to charts, allowing users to create [custom visualizations for each state](https://mui.com/x/react-charts/styling/#overlay)
+- 📝 Fixed a typo in an undocumented [x-charts gradient id](https://github.com/mui/mui-x/pull/12941/files/4bf4bffbc2799a01a96bc7458a17318cf41c1722#diff-26c31cc69d6f51110f89e339578ef9b3d4a3551f79077fff73f7babb81c5099f) that could have unintended side effects
```

Comment:
Nitpick: Is it worth adding such a thing into highlights? 🤔

Reply:
I was unsure where to put it, moved it to just below the commit

Reply:
Oh, sorry, I didn't check what is all it about. 🙈 
Yeah, in that case, I think that mentioning it in highlights makes much more sense...
Sorry for the confusion. 🤷 
I'd maybe rephrase it to be more specific, targetting the actual problem - change in the id of an element instead of saying "unintended side effects", which sounds more scary to me, when reading about it. 🙈

Reply:
done